common/cmake/devices.cmake

```diff
@@ -1381,6 +1566,11 @@ function(ADD_FPGA_TARGET)
     set(OUT_CONSTR_REL ${OUT_LOCAL_REL}/${TOP}_constraints.place)
     set(OUT_NET ${OUT_LOCAL}/${TOP}.net)
 
+    get_target_property(PLACE_TOOL_EXTRA_ARGS ${BOARD} PLACE_TOOL_EXTRA_ARGS)
```

Comment:
Not sure why this is here?

Reply:
Remnant of old code I was using for placement constraints.  Removing.